chimney/src/main/scala-2/io/scalaland/chimney/internal/compiletime/derivation/LegacyPlatform.scala

```diff
@@ -0,0 +1,41 @@
+package io.scalaland.chimney.internal.compiletime.derivation
+
+import io.scalaland.chimney.internal.compiletime.{ConfigurationsPlatform, Contexts, DefinitionsPlatform}
+import io.scalaland.chimney.internal.macros.dsl.TransformerBlackboxMacros
+import io.scalaland.chimney.{PartialTransformer, Transformer}
+
+private[compiletime] trait LegacyPlatform extends Legacy {
+  this: DefinitionsPlatform & ConfigurationsPlatform & Contexts =>
+
+  protected object legacy extends LegacyImpl {
+
+    private def convertToLegacyConfig[From, To](
+        tbm: TransformerBlackboxMacros
+    )(implicit ctx: Context.ForTransformer[From, To]): tbm.TransformerConfig = {
+      // TODO: convert our new config into our old config
+      ???
```

Comment:
I intend this to use values from Context (including `ctx.config.legacy`) to generate `TransformerConfig` and pass it to original macro generating code.